xml_analyzer/__init__.py

```diff
@@ -1,6 +1,8 @@
-from .calculator import Calculator
+from .data_container import DataContainer
 from .xmlreader import XMLReader
 from .formula import Formula
+from .definitions import AutocalcMethods, EquationTypes
+from .calculator import Calculator
 
-__all__ = ['Calculator', 'XMLReader', 'Formula']
+__all__ = ['Calculator', 'XMLReader', 'Formula', 'AutocalcMethods', 'EquationTypes', 'DataContainer']
```

Comment:
why is every module from this package publicly exposed. Only Calculator and XMLReader is used in main

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/f1272272794129d7e2ff4e59f447510111bf27f4